CRM/Report/Form/Activity.php

```diff
@@ -698,6 +714,27 @@ public function add2group($groupID) {
     }
   }
 
+  /**
+   * @param $fields
+   * @param $files
+   * @param $self
+   *
+   * @return array
+   */
+  public static function formRule($fields, $files, $self) {
+    $errors = array();
+    $componentId = CRM_Core_Component::getComponentID('CiviCase');
+    $caseActivityTypes = CRM_Core_OptionGroup::values('activity_type', TRUE, FALSE, FALSE," AND v.component_id={$componentId}");
```

Comment:
@jitendrapurohit  Please add a space after the comma and before the double quotes on this line.